mason.sh

```diff
@@ -86,6 +86,19 @@ elif [ ${MASON_PLATFORM} = 'ios' ]; then
     export MASON_ISIM_CFLAGS="${MIN_SDK_VERSION_FLAG} -isysroot ${MASON_SDK_PATH} -arch i386 -arch x86_64"
 
 elif [ ${MASON_PLATFORM} = 'linux' ]; then
+    MASON_CXX=${CXX:-`c++ -v 2>&1`}
+
+    if [[ ${CXX} ]]; then
+        MASON_CXX_DUMPVERSION=`${CXX} -dumpversion`
+    else
+        MASON_CXX_DUMPVERSION=`c++ -dumpversion`
+    fi
```

Comment:
Can we shorten this block to `${CXX:-c++} -dumpversion` and directly use it below?